app/assets/stylesheets/generic/flash.scss

```diff
@@ -1,9 +1,10 @@
 .flash-container {
   cursor: pointer;
-    margin: 0;
-    font-size: 14px;
-    width: 100%;
-    z-index: 100;
+  margin: 0;
```

Comment:
Properties should be ordered cursor, font-size, margin, position, width, z-index